### DISCUSSION TITLE
SimToolkitPro icon

### DISCUSSION THREAD

**Discussion by dferdo61** [elrumo/macOS_Big_Sur_icons_replacements#1316] [Icon request]
https://simtoolkitpro.co.uk/

Original:
<img width="512" alt="SimToolkitPro" src="https://user-images.githubusercontent.com/22680916/139039235-f93f2008-8b50-43ad-9f07-3d2adf0c0fa3.png">

**Comment by ThaliaMyers** [comment]
Done. It is on my profile waiting for approval.
https://macosicons.com/u/CaedmonM

**Reply by dferdo61** [reply]
Thanks!
